maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/AbstractSurefireMojo.java

```diff
@@ -1560,6 +1566,18 @@ private void convertGroupParameters()
         }
     }
 
+    private void convertJunitEngineParameters()
+    {
+        if ( this.getJunitIncludeEngine() != null )
```

Comment:
I think this class is using `isNotBlank` somewhere too. It is in one of the classes `StringUtils`/ It is very useful because it has the same meaning for `not` and empty strings or strings having white spaces, since they do not cover any reasonable value, and the method returns `false` in these cases.
Same in the second IF below.